tools/SetupFlow/DevHome.SetupFlow/Models/CreateDevDriveTask.cs

```diff
@@ -43,7 +44,7 @@ public IDevDrive DevDrive
         get; set;
     }
 
-    public CreateDevDriveTask(IDevDrive devDrive, IHost host, ISetupFlowStringResource stringResource)
+    public CreateDevDriveTask(IDevDrive devDrive, IHost host, SetupFlowOrchestrator setupFlowOrchestrator, ISetupFlowStringResource stringResource)

```

Comment:
Should this also take a Guid as input to be consistent?

Reply:
True.  True.  Changed.